runtime/jvm/src/main/kotlin/kotlinx/serialization/java/JavaTypesModule.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.java
+
+import kotlinx.serialization.*
+import kotlinx.serialization.context.*
+import java.math.*
+import java.net.*
+import java.util.*
+import java.util.concurrent.atomic.*
+import kotlin.reflect.*
+
+/**
+ * Module with external serializers for the following Java types:
```

Comment:
How are you going to combat this class from playing whack-a-mole forever in the future from people wanting their type included. For example, why isn't `AtomicIntegerArray` included? Why isn't `java.time.*`. Why isn't `File`?

Reply:
I don't mind including other types on a demand-driven basis when it is reasonable, an additional bonus is that this kind of features is a good "first-time contribution".

The only question is what is a "reasonable" request. E.g. `java.sql.Date` is a good candidate, while `java.io.File` is definitely not (at least because its representation is OS/machine/JVM specific). I know no strict criteria for a type to be included except common sense.